src/main/java/com/coveo/configuration/parameterstore/ParameterStorePropertySourceConfigurationProperty.java

```diff
@@ -0,0 +1,19 @@
+package com.coveo.configuration.parameterstore;
+
+public final class ParameterStorePropertySourceConfigurationProperty
+{
+    private static final String PREFIX = "awsParameterStorePropertySource.";
+    private static final String SSM_CLIENT_PREFIX = "awsParameterStoreSource.ssmClient.";
+
+    public static final String ACCEPTED_PROFILE = "awsParameterStorePropertySourceEnabled";
+
+    public static final String ENABLED = PREFIX + "enabled";
+    public static final String ACCEPTED_PROFILES = PREFIX + "enabledProfiles";
```

Comment:
I would use String.join("." everywhere, instead of having the "." in the constants.